autoendpoint/src/extractors/notification_headers.rs

```diff
@@ -7,7 +7,7 @@ use lazy_static::lazy_static;
 use regex::Regex;
 use std::cmp::min;
 use std::collections::HashMap;
-use validator::Validate;
+use validator::{Validate, ValidationError};
```

Comment:
Again, we want to isolate the urgency stuff to a feature flag, so the bits that are urgency based need to be behind that flag.